src/features/user/Settings/EditProfile/EditProfileForm.tsx

```diff
@@ -296,150 +321,225 @@ export default function EditProfileForm() {
   };
   let suggestion_counter = 0;
 
+  const EditProfileInputContainer = styled('div')({
+    flex: 1,
+    minWidth: 180,
+    display: 'flex',
+    flexDirection: 'column',
+    gap: 8,
+  });
```

Comment:
This could be used as a common component, so should be placed in the `common` folder, and also given a more generic name e.g. `FormFieldContainer` or `FormInputContainer`

Reply:
This probably isn't needed any more, as we're sticking with the old field styles that are used in the rest of the dashboard/app